### DISCUSSION TITLE
Welcome to MISP Discussions!

### DISCUSSION THREAD

**Discussion by adulau** [MISP/MISP#8009] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WM-2028** [comment]
I am very new to MISP, I have been asked to get it stood up and see if it would be a benefit to us. I was able to download an Image file from Circle and get it up and running. I am a Security Analyst and have had some scripting classes but am far from being a developer. Other than the basics as far as changing the initial password when I first logged in, I can't seem to find any documentation as to what my next step is. I have found some interesting and useful videos but a lot of the options they show I do not see in my instance of MISP. Such as all the Enhancements available when you want to enhance an event. At the bottom of my console I see " This is an initial install Powered by MISP 2.4.168 Please configure and harden accordingly - 2023-02-17 15:42:50" but don't know what I need to do to configure and harden it. Can someone point me in the right direction as to what I need to do next? If I am in the wrong spot, then please let me know where I should be asking for this help.

**Comment by WM-2028** [comment]
I am a Security Analyst and hope to get MISP working so that I can contribute something to this community.
